### PR TITLE
VB-3331 Update UI for cancelled count, added some extra bit's of info…

### PR DESCRIPTION
**Pull request by AledWEvans**
added some extra bit's of info for the admin service, cancel info

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt [hunk 1]
```diff
@@ -83,6 +79,18 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
     @Param("visitsToDate") visitsToDate: LocalDate?,
   ): List<VisitCountsByDate>
 
+  @Query(
+    "select count(*) from visit v " +
+      " WHERE v.session_template_reference = :reference" +
+      "   AND v.visit_start >= :sessionDate AND v.visit_start <= :sessionDate" +
+      "   AND visit_status IN ('CANCELLED') AND v.outcome_status != 'SUPERSEDED_CANCELLATION'",
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt:86]: I think we need to add v.outcome_status IS NULL to v.outcome_status != 'SUPERSEDED_CANCELLATION' like the query above (line 56)

**Review by tpmcgowan — DISMISSED** [review]
LGTM!

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt (hunk 1 above)
Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt]: Already done,... just did not push up the correct code,...

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request merged**